lib/screens/login_page.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:badiup/screens/customer_home_page.dart';
+import 'package:flutter/material.dart';
+
+import 'package:badiup/sign_in.dart';
+import 'package:badiup/screens/admin_home_page.dart';
+
+class LoginPage extends StatefulWidget {
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        color: Colors.white,
+        child: Center(
+          child: Column(
+            mainAxisSize: MainAxisSize.max,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              Text(
+                'BADI UP', 
+                style: TextStyle(fontSize: 30.0),
+              ),
+              SizedBox(height: 50),
+              _buildLoginButton(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildLoginButton() {
```

Comment:
I'd reduce this further into smaller methods!

Reply:
Let me refine this page after the login page's design is decided.
Now just keep this.